### DISCUSSION TITLE
Is there a way to upload something like Vue component libraries to JSR?

### DISCUSSION THREAD

**Discussion by mybearworld** [jsr-io/jsr#753] [Q&A]
Vue files use their own syntax with `.vue`, so they need to be built. NPM is fine with .js and .d.ts files which can be generated by Vite and `vite-plugin-dts`. This is discouraged by JSR, however, with the reasoning that you can just upload Typescript files instead. So what should you do for Vue files?

**Comment by golsch** [comment]
Hello, do you have any new insights into this?
